sources/csharp2/KalturaClient/Types/ObjectBase.cs

```diff
@@ -132,10 +132,16 @@ protected int ParseInt(string s)
         protected Single ParseFloat(string s)
         {
             Single i = Single.MinValue;
-            Single.TryParse(s, out i);
+            Single.TryParse(s, out i);            
             return i;
         }
-        
+        protected double ParseDouble(string s)
+        {
+            double i = Double.MinValue;
+            Double.TryParse(s, out i);
+            return i;
```

Comment:
that code seems wrong, but i see its a mistake that was made long before you touched the code . 
i will never get Double.MinValue .. since tryParse has "out" param it wil always overwrite with what TryPArseReturns which is 0 in case of failure to parse :) .. so proper code should look like 

```csharp
if (Double.TryParse(s, out var parsedNum){
  return parsedNum
}
return Double.MinValue;

```

Reply:
we won't change this is the behavior in other client properties